pom.xml

```diff
@@ -191,6 +192,13 @@
       <groupId>io.jenkins.configuration-as-code</groupId>
       <artifactId>test-harness</artifactId>
       <scope>test</scope>
+      <!-- TODO https://github.com/jenkinsci/bom/pull/1011 -->
+      <exclusions>
+        <exclusion>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>jackson2-api</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Only temporary until https://github.com/jenkinsci/bom/pull/1011 is resolved.

I could go to test-harness and attempt to make another PR but it would be yet another PR that blocks bom...